pygmt/datasets/moon_relief.py

```diff
@@ -0,0 +1,93 @@
+"""
+Function to download the Moon relief datasets from the GMT data server,
+and load as :class:`xarray.DataArray`.
+
+The grids are available in various resolutions.
+"""
+from pygmt.datasets.load_remote_dataset import _load_remote_dataset
+from pygmt.helpers import kwargs_to_strings
+
+__doctest_skip__ = ["load_moon_relief"]
+
+
+@kwargs_to_strings(region="sequence")
+def load_moon_relief(resolution="01d", region=None, registration=None):
+    r"""
+    Load Moon relief in various resolutions.
+
+    The grids are downloaded to a user data directory
+    (usually ``~/.gmt/server/moon/moon_relief/``) the first time you invoke
+    this function. Afterwards, it will load the grid from the data directory.
+    So you'll need an internet connection the first time around.
+
+    These grids can also be accessed by passing in the file name
+    **@moon_relief**\_\ *res*\[_\ *reg*] to any grid plotting/processing
+    function. *res* is the grid resolution (see below), and *reg* is grid
+    registration type (**p** for pixel registration or **g** for gridline
+    registration).
+
+    The default color palette table (CPT) for this dataset is *@moon_relief.cpt*.
+    It's implicitly used when passing in the file name of the dataset to any
+    grid plotting method if no CPT is explicitly specified. When the dataset
+    is loaded and plotted as an :class:`xarray.DataArray` object, the default
+    CPT is ignored and GMT's default CPT (*turbo*) is used. To use the
```

Comment:
```suggestion
    CPT is ignored, and GMT's default CPT (*turbo*) is used. To use the
```

Reply:
In case with add this comma, I can add this change for the other dataset in PR #2688.